app/services/action_handler.rb

```diff
@@ -1,4 +1,6 @@
 class ActionHandler < BaseService
+  private_constant :ACTIONS
```

Comment:
This will break the tests. :(

This change doesn't have the intended effect. We need to keep those `private_constant`s in place.